lib/measures/ladybug_energy_model_measure/tests/example_face_by_face_model.json

```diff
@@ -1,15 +1,19 @@
 {
     "convert_to_meters": 1,
-    "type": "FaceByFaceModel",
+    "type": "Model",
```

Comment:
@tanushree04 seems like we might want two models in the measure tests, a maximal one will all the energy model stuff defined and a minimal one without any of the energy model stuff.  We should rename the test files to reflect what they are, there is a special git rename command to rename a file once it is in git: https://help.github.com/en/articles/renaming-a-file-using-the-command-line